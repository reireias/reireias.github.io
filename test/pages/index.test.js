import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/pages/index'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('index', () => {
  it('should be vue instance', () => {
    const wrapper = shallowMount(target, {
      localVue,
      Vuetify
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
