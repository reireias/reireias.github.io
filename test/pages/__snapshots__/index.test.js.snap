// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should be vue instance 1`] = `
"<div><top-stub style=\\"margin-top: 100px; margin-bottom: 100px;\\"></top-stub> <div class=\\"title-text\\">
    Profile
  </div> <profile-stub></profile-stub> <div class=\\"title-text\\">
    Account
  </div> <account-stub></account-stub> <div class=\\"title-text\\">
    Certificate
  </div> <certificate-stub></certificate-stub> <div class=\\"title-text\\">
    Skills
  </div> <hexagon-skill-stub></hexagon-skill-stub> <div class=\\"title-text\\">
    Career
  </div> <timeline-stub></timeline-stub></div>"
`;
